R/Analysis.R

```diff
@@ -224,5 +226,13 @@ MakeEnrichmentDotPlot <- function(seuratObj,
     ylab(yField) +
     guides(fill = guide_colorbar(order = 1))
   
+  if(!is.null(facetingField)){
+    if(!(facetingField %in% extraGroupingFields)){
+      stop("Please add your desired facetingField to the extraGroupingFields variable.")
+    }
+    P1 <- P1 + 
+      facet_wrap(as.formula(paste("~", facetingField)), scales = "free_x")
```

Comment:
we're making the assumption that we want horizontal faceting. is there a rational way to make this more flexible? Could they provide a vector of facet fields (like c(".", "myField") vs.  c("myField", ".") vs. c("myField1", "myField2")?

Reply:
Circling back around on this: after doing some analysis on PC475 a couple of weeks ago, I agree that pre-prescribing the faceting is more trouble than it's worth and reduces flexibility too much. I'll close this PR.